Common/include/option_structure.hpp

```diff
@@ -1113,6 +1113,7 @@ enum class SA_OPTIONS {
   ROT,      /*!< \brief Rotation correction. */
   BC,       /*!< \brief Bas-Cakmakcioclu transition. */
   EXP,      /*!< \brief Allow experimental combinations of options (according to TMR). */
+  PGOMGA,   /*!< \brief Use PGOMGA term. */
```

Comment:
Please add the DOI address of your reference somewhere (maybe here), i.e. some `https://doi.org/<the DOI string>`address